samza-test/src/main/config/join/joiner.samza

```diff
@@ -16,6 +16,33 @@
 # specific language governing permissions and limitations
 # under the License.
 
+####################
+### UPDATE THIS! ###
```

Comment:
Can you add more details here? Not sure what update this refers to?

Reply:
These unit tests are reading two files, common.properties and joiner.samza, as the input config, after updating to use ConfigLoader, we do not want to read from multiple files anymore, so I am deleting common.properties file and move all its properties in each job config file instead.